src/main/java/com/stripe/model/PersonVerificationDeserializer.java

```diff
@@ -0,0 +1,88 @@
+package com.stripe.model;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonPrimitive;
+
+import java.lang.reflect.Type;
+
+public class PersonVerificationDeserializer implements JsonDeserializer<Person.Verification> {
+  /**
+   * Deserializes a person.verification JSON payload into a {@link Person.Verification} object.
+   */
+  @Override
+  public Person.Verification deserialize(JsonElement json, Type typeOfT,
+      JsonDeserializationContext context) throws JsonParseException {
+    Gson gson = new GsonBuilder()
+        .setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+        .registerTypeAdapter(ExpandableField.class, new ExpandableFieldDeserializer())
```

Comment:
Why don't you need to register all deserializers?

Reply:
We only use this instance to deserialize `Person.Verification`, `Person.VerificationDocument` and `File`, so we don't need any of the other custom deserializers. (Ideally we'd always use the global deserializer instance in `ApiResource`, but then that'd cause an infinite recursion problem, cf. #433).